Lab2.ipynb

```diff
@@ -0,0 +1,830 @@
+      "text/plain": [
+       "<Figure size 720x504 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.rcParams[\"figure.figsize\"] = (10,7)\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "d1 = stats.poisson.pmf(x, mu)\n",
+    "n = 200\n",
+    "for i in range(1, n):\n",
+    "    \n",
+    "    dist_200 = np.convolve(d1,d2)\n",
+    "    d1 =  dist_200\n",
+    "\n",
+    "ax.plot(dist_200);\n",
+    "plt.xlabel('number of cosimis ray counted', fontsize=18)\n",
+    "plt.ylabel('probability', fontsize=18);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Q1B\n",
+    "Here you can see that by changing the summing the probability for 5 days, the distribution is still a poisson distribution. This is the unique property of poisson distribution. The more numbers we convolve the higher the mean value of the final distribution. This is because as we convolve the two distribution we are ultimately summing the two independent random variables from the two distributions so the mean value increases."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Q1 C\n",
```

Comment:
It will be great if you could label the axes and add titles for all the plots in this section.